driver-webdriver/index.js

```diff
@@ -28,8 +28,11 @@ async function mochifyDriver(options = {}) {
   await client.navigateTo(url || default_url);
 
   return {
-    evaluate: (script) => client.executeScript(script, []),
-    evaluateReturn: (script) => client.executeScript(`return ${script}`, []),
+    evaluate: (script) =>
+      client.executeScript(
+        `return (function () { var __mochify_return_value = ${script}; return __mochify_return_value; })()`,
```

Comment:
I'm not entirely sure if the wrapping IIFE is even needed. This also works if you do

```js
var __mochify_return_value = ${script}; return __mochify_return_value;
```

yet I have a hard time understanding if each `executeScript` call creates a function scope by itself or not and if this could create possible collisions.

See: https://w3c.github.io/webdriver/#dfn-execute-script

Reply:
Thinking about it, if it's ok to use a `return` statement, this must be run in the context of a function, so it's probably fine to omit the wrapper.

Reply:
AFAIK some implementations warp the code in a function and some don't.

We also use this to inject the mochify client, which I guess works because that is wrapped in a function too.

I'll run some tests as well this weekend.